src/test/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationServiceTest.java

```diff
@@ -864,10 +929,77 @@ void shouldFilterFieldsWithinCollectionItems() throws IOException {
             );
         }
 
+        @Test
+        // TODO Target implementation, see RDM-1204
+        @DisplayName("should filter out collection item if no item classification exists")
```

Comment:
New tests that were missing for SecurityClassificationService to increase branch coverage.